client/components/gravatar/test/index.jsx

```diff
@@ -1,96 +1,107 @@
 /**
  * External dependencies
  */
-import assert from 'assert';
-import ReactDomServer from 'react-dom/server';
+import { expect } from 'chai';
 import React from 'react';
+import { shallow } from 'enzyme';
 
 /**
  * Internal dependencies
  */
 import { Gravatar } from '../';
 
-/**
- * Pass in a react-generated html string to remove react-specific attributes
- * to make it easier to compare to expected html structure
- * @param {string} string react-generated html
- * @return {string} HTML without react attributes
- */
-function stripReactAttributes( string ) {
-	return string.replace( /\sdata\-(reactid|react\-checksum)\=\"[^\"]+\"/g, '' );
-}
+describe( 'Gravatar', () => {
+	/**
+	 * Gravatar URLs use email hashes
```

Comment:
could we also add in this comment _what_ is being hashed? could be useful to know the actual email address